js/ui/text_box/ui.text_editor.base.js

```diff
@@ -331,6 +334,14 @@ const TextEditorBase = Editor.inherit({
         });
     },
 
+    _compressInputContainer: function() {
+        const $element = this.$element();
+        const isIE11 = browser.msie && browser.version <= 11;
+        if(isIE11 && $element.css('display') === 'block') {
+            $element.addClass(TEXTEDITOR_COMPRESSED_CLASS);
```

Comment:
$element.css('flexBasis', 0)?

Reply:
It's better to avoid the use of inline styles - they are more difficult to override if necessary.

Reply:
Make sense. Maybe we can come up with a better name for the CSS class.